src/main/java/connor/ui/Ui.java

```diff
@@ -0,0 +1,58 @@
+package connor.ui;
+
+import connor.task.Task;
+
+public class Ui {
+    public static final String LINE = "        ________________________________________________________\n";
+
+    public static void printMessage(String response) {
+        System.out.println(LINE + "        " + response);
```

Comment:
I feel that the empty spaces can be extracted as a constant variable and appended to the output strings, instead of adding the spaces in front like this. May have inconsistent number of spaces too!